pangolp/mod-npc-promotion
# feat. Upgrade module to the latest emulator version

## Changes Proposed:
-  Move the items to the database, to make it easier to modify them.
-  Update the module structure to be compatible with the latest version of the emulator.

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- In-game
- Windows 10


## How to Test the Changes:


1.When using the code, all classes should be tested, and if any of them fail, then we should update the script. The items depend on the database values. The default set is set 78. However, it can be edited in the database.
